client/src/Components/SearchPage/CardResultFilter/Card.jsx

```diff
@@ -17,7 +17,7 @@ export default function Card({ filter }) {
   return (
     <div className="card__filter">
       <div className="image__card">
-        <img alt="complex" src={images[0].pathUrl} />
+        {images.length && <img alt="complex" src={images[0].pathUrl} />}
```

Comment:
I think it's better to render logo if images length equals zero
```suggestion
        {images.length ? <img alt="complex" src={images[0].pathUrl} /> : <img alt="complex" src={logo // or whatever} />  }
```